README.md

```diff
@@ -89,8 +89,8 @@ from qiskit.algorithms.optimizers import L_BFGS_B
 optimizer = L_BFGS_B()
 
 # setup the mapper and qubit converter
-from qiskit_nature.mappers.second_quantization import ParityMapper
-from qiskit_nature.converters.second_quantization import QubitConverter
+from qiskit_nature.second_quantization.operators.fermionic import ParityMapper
```

Comment:
Some thoughts. While `second_quantization` was there before this import is starting to get long and has pulled up the mappers into the fermionic folder to be accessed from there. For the former I am wondering if using `second_q` for the folder would suffice - we use it elsewhere like second_q_ops() - the problem method etc. So `FermionicOp` will be imported from `operators` since they are all imported up to there and documented there and mappers from the operator subfolders, like fermionic. For the latter part of the initial comment, seeing mappers in the path in the past the import and folder seemed to be more connected context wise. I get that these mappers are for fermionic operators but I am not importing an operator from there rather a mapper. It just feels a little off - maybe its the newness/difference from before, I don;t know...

Reply:
I agree with your suggestion on `second_q`. Let's go with that :+1: 

On the notion of from where to import I do not have a strong opinion. But these are also some details we can iron out later :+1: